cpp/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ if(NOT CMAKE_CUDA_COMPILER)
   message(SEND_ERROR "CMake cannot locate a CUDA compiler")
 endif(NOT CMAKE_CUDA_COMPILER)
 
+include(cmake/Modules/ccache.cmake)
```

Comment:
Can this be guarded by a CMake option? Changing the CMake compiler launcher will break anyone using ccache compiler rsymlinks.

Reply:
Done.